sources/platform/actors/running/usage_and_resources.md

```diff
@@ -13,25 +13,35 @@ slug: /actors/running/usage-and-resources
 
 ## Resources
 
-[Actors](../index.mdx) run in
-[Docker containers](https://www.docker.com/resources/what-container),
-which have a [limited amount of resources](https://phoenixnap.com/kb/docker-memory-and-cpu-limit) (memory, CPU, disk size, ...). When starting, the Actor needs to be allocated a certain share of those resources, such as how much CPU "horsepower" it will be able to use.
+[Actors](../index.mdx) run in [Docker containers](https://www.docker.com/resources/what-container), which have a [limited amount of resources](https://phoenixnap.com/kb/docker-memory-and-cpu-limit) (memory, CPU, disk size, etc). When starting, the Actor needs to be allocated a certain share of those resources, such as CPU capacity that is necessary for the Actor to run.
 
 ![Setting an Actor's memory](./images/usage_and_resources/memory-settings.png)
 
-By assigning an Actor's **memory** capacity, you also assign it a proportional amount of **CPU power**. In addition to CPU power, the memory allocation also influences disk size, which amounts to 2x the allocated memory.
+By assigning an Actor's **Memory** capacity, you also assign it a proportional amount of CPU power. The memory allocation also influences the disk size, which amounts to 2x the allocated memory.
```

Comment:
Suggestion: Maybe I'd change this to saying that setting memory also sets CPU and disk size, and then just have it described in the respective sections.